policyengine_us/parameters/gov/states/mt/tax/income/main/head_of_household.yaml

```diff
@@ -0,0 +1,59 @@
+description: Montana taxes head of household filers at this rate structure.
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: Montana income tax rates for head of household filers
+  reference:
+    - title: Montana Code Annotated 2021 Title 15, Chapter 30, Part 21, 15-30-2103 
+      href: https://leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0030/0150-0300-0210-0030.html
+# Before 2024 Montana has a flat tax rate for each filing status
```

Comment:
Not a flat rate right, just the same across filing statuses?

Reply:
correct, I changed the formatting